server/db/seed_db.ts

```diff
@@ -11,47 +11,37 @@ import { AdminRepository } from "../src/repositories/admin_repository";
 (async () => {
     const pool = new Pool();
     const userRepository = new UserRepository(pool);
+    const doctorRepository = new DoctorRepository(pool, userRepository);
+    const patientRepository = new PatientRepository(pool, userRepository);
+    const adminRepository = new AdminRepository(pool, userRepository);
 
     // add addresses
-    await Promise.all(
-        addresses.map(async (address) => {
-            await userRepository.addAddress(address);
-        }),
-    );
-
-    console.log("Finished seeding addresses");
+    for (const address of addresses) {
+        await userRepository.addAddress(address);
+    }
 
     // add users
-    await Promise.all(
-        users.map(async ({ userData, addressId }) => {
-            userData.password = await bcrypt.hash(userData.password, 10);
-            await userRepository.addUser(userData, addressId);
-        }),
-    );
+    for (const { userData, addressId } of users) {
```

Comment:
This will make it so the userid and the email number line up until we refactor the seed data to include a bulk amount

Reply:
Makes sense